src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java

```diff
@@ -317,15 +545,36 @@
  * Example of allowed fields:
  * </p>
  * <pre>
- * class SomeClass
- * {
- *   &#64;CustomAnnotation
- *   String customAnnotated; // no warning
- *   &#64;com.annotation.CustomAnnotation
- *   String customAnnotated1; // no warning
- *   &#64;mypackage.annotation.CustomAnnotation
- *   String customAnnotatedAnotherPackage; // another package but short name matches
- *                                         // so no violation
+ * public final class MyClass {
+ * private int myPrivateField1;
+ *
+ * int myPrivateField3;               // violation, must have a visibility modifier
+ *
+ * protected String myProtectedField; // violation, protected visibility is not allowed
+ *
+ * public int NOT_CONSTANT = 42;     // violation
+ *                                  // not static final, immutable, matching the pattern
+ * public long serialVersionUID = 123456789L;
+ * public static final int MY_CONSTANT = 42;
+ *
+ * public final int MY_CONSTANT2 = 42;                            // violation
+ *                                                   // public immutable fields are not allowed
+ *
+ * public final java.lang.String notes;                           // violation
+ *                                                  // public immutable fields are not allowed
+ *
+ * public final Set&lt;String&gt; mySet1 = new HashSet&lt;&gt;(); // violation, HashSet is mutable
+ * public final ImmutableSet&lt;String&gt; mySet2;                // violation, mutable
+ * public final ImmutableMap&lt;String, Object&gt; objects1;      // violation, mutable
+ *
+ * &#64;com.annotation.CustomAnnotation
+ * String annotatedString;                                        // violation
```

Comment:
collection lines are too long, but it force you to have `// violation` too far on right. Lets not align all by such violation.
Make two alignments: for long lines and for short lines 

```
 * public final int AGE = 42;           // violation
 * public final java.lang.String notes; // violation
 * public final Set&lt;String&gt; mySet1 = new HashSet&lt;&gt;(); // violation, HashSet is mutable
```

Reply:
done